testing/fwtests/dbtest1.js

```diff
@@ -0,0 +1,11 @@
+function(context, args) {
+
+    var k = "akey";
+    var o = { obj:"obj" };
+
+    var i = #db.i({ k: k, o: o });
+
+    var o2 = #db.f({ k: k }).array();
+    console.log(o2);
```

Comment:
Doesn't actually work in hackmud, FYI. (`console` doesn't exist, so there's no `console.log`)